### PR TITLE
Add: support more than one CDN url

### PR DESCRIPTION
**Pull request by TrueBrain**
This is useful to distribute load/cost over multiple servers.

### CODE REVIEW

**Comment by TrueBrain** [comment]
@LordAro @frosch123 : could either of you review this? Cheers!

**Review by LordAro — APPROVED** [review]

File: bananas_server/web_routes.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
+import aiohttp
```
[the remark this answers is not in the document]
Comment (TrueBrain) [reply bananas_server/web_routes.py:1]: This is such a baloni thing of CodeQL to mention .. I already put it on "won't fix". But okay, I will fix it ... it doesn't increase maintainability, but what-ever :P Some things CodeQL do are just bull :P

[the remark this answers is not in the document]
Comment (TrueBrain) [reply bananas_server/web_routes.py:1]: no, I take that back. I am not going to fix it :P ` async with ClientSession() as session:` is completely unreadable :)

File: bananas_server/web_routes.py [hunk 2]
```diff
@@ -12,7 +15,35 @@
...
 BANANAS_SERVER_APPLICATION = None
+CDN_FALLBACK_URL = None
 CDN_URL = None
+CDN_ACTIVE_URL = []
+
+
+async def check_cdn_health():
+    await asyncio.sleep(1)
+
+    log.info("Healthchecks for CDN servers %r enabled", CDN_URL)
```
[the remark this answers is not in the document]
Comment (TrueBrain) [reply bananas_server/web_routes.py:26]: yes, `%r`. CDN_URL is a list or `None`, so using `__repr__` gives the best readable result, without doing complex things. Not sure what your question is here honestly :)

[the remark this answers is not in the document]
Comment (LordAro) [reply bananas_server/web_routes.py:26]: Just didn't recognise it

**Pull request merged**